semperfiwebdesign/all-in-one-seo-pack
# Do not noindex/nofollow paginated archives

Issue #906

## Proposed changes

Fixes a bug where the "noindex/nofollow for paginated posts/pages" settings also affect archive pages.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

1. Enable the "noindex/nofollow settings for paginated posts/pages" in the General Settings menu.
2. Create a paginated post, paginated page and paginated CPT.
3. Make sure that all of these are correctly noindexed/nofollowed.
4. Make sure that archive pages they should appear on (date archives, author archives, etc.) are unaffected. You might need to add additional Posts/Pages/CPTs before their archive pages are paginated.

## Further comments

You'll notice from the code changes that this simply adds an additional is_singular() check.
